### PR TITLE
Update composer.json author

### PR DESCRIPTION
**Pull request by le-melon**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | https://www.usine-digitale.fr/article/l-editeur-de-sites-e-commerce-prestashop-rachete-par-mbe-worldwide.N1163817
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28100)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "license": "OSL-3.0",
     "authors": [
         {
-            "name": "PrestaShop SA",
-            "email": "contact@prestashop.com"
+            "name": "MBE Worldwide",
+            "email": "contact@mbecorporate.com/"
```
Comment (PierreRambaud) [inline composer.json]:
```suggestion
            "email": "contact@mbecorporate.com"
```
sorry, incorrect email.

@matks wdyt?

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
The email seems incorrect, dunno if we can accept it

**Review by PrestaEdit — CHANGES_REQUESTED** [review]
It's sill PrestaShop SA, as I know.
Also, it's maybe not more PrestaShop SA nor MBE, it's PrestaShop OS Project, imho.

**Comment by le-melon** [comment]
Integration tests, PHP / PHP CS Fixer and Sanity failing

need to fix it I will open a not break PR

**Comment by eternoendless** [comment]
Ha ha, nice April fool's joke ;)

**Pull request closed without merging**